### PR TITLE
OSDOCS-8352: Z-stream release notes for 1.4.2 CVE fix

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.11+

Issue:

https://issues.redhat.com/browse/OSDOCS-8352
https://issues.redhat.com/browse/NETOBSERV-1371
Link to docs preview:

https://66890--docspreview.netlify.app/openshift-enterprise/latest/network_observability/network-observability-operator-release-notes

NOTE: In the preview Errata link will be updated when the live link is available. 
QE review:
- [x] QE has approved this change.

Additional information:

1.4.2 release date not yet confirmed. Likely sometime near 11/8.

### CODE REVIEW

File: network_observability/network-observability-operator-release-notes.adoc [hunk 1]
```diff
@@ -12,6 +12,15 @@ The Network Observability Operator enables administrators to observe and analyze
 These release notes track the development of the Network Observability Operator in the {product-title}.
 
 For an overview of the Network Observability Operator, see xref:../network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
+[id="network-observability-operator-release-notes-1-4-2"]
+== Network Observability Operator 1.4.2
+The following advisory is available for the Network Observability Operator 1.4.2:
+//NOTE: this link will be broken until the live errata is available on 1.4.2 GA, 10/30/23
+* link:https://access.redhat.com/errata/RHSA-2023:122668[2023:122668 Network Observability Operator 1.4.2]
```
Comment (stleerh) [inline network_observability/network-observability-operator-release-notes.adoc:19]: Need a blank line before line 19 so it puts the bullet item on the next line

Comment (stleerh) [inline network_observability/network-observability-operator-release-notes.adoc:19]: Need a blank line before line 19 so it puts the bullet item on the next line

Comment (skrthomas) [reply network_observability/network-observability-operator-release-notes.adoc:19]: Thanks Steven. I noticed this and figured I'll fix it when I get the live ID for errata.

**Comment by bscott-rh** [comment]
Peer review LGTM.

**Review by bscott-rh — APPROVED** [review]

**Comment by skrthomas** [comment]
https://github.com/openshift/openshift-docs/pull/67567

fixing the errata link

**Pull request merged**